.github/workflows/api-pull-request.yml

```diff
@@ -62,6 +62,7 @@ jobs:
             - name: Run Tests
               env:
                   DATABASE_URL: "postgres://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/postgres"
+                  ANALYTICS_DATABASE_URL: "postgres://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/postgres"
```

Comment:
A couple of points here: 

 * Should we at least use a different database here? 
 * Does the migration process by default migrate all databases?

Reply:
> Does the migration process by default migrate all databases?


Pyest does